src/Commands/HttpServerCommand.php

```diff
@@ -370,7 +370,7 @@ protected function checkEnvironment()
             exit(1);
         }
 
-        if (! extension_loaded('swoole')) {
+        if (! extension_loaded('swoole') && ! extension_loaded('openswoole')) {
```

Comment:
shouldn't this be an or statement?

```php
...
if (! extension_loaded('swoole') or ! extension_loaded('openswoole')) {

...

```

Reply:
@Arkanius thanks for your reply. Nope it will be `and` operator. bcoz Sometimes one of the two may be available but it does not mean that one of them missing means others is not available. when both of them are missing then we should show the message.